Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml

```diff
@@ -516,7 +519,133 @@ script:
+      description: The event ID to retrieve the forensics analyzers from
+      isArray: false
+      name: event_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve Illusive's forensics analyzers on a certain event
+    execution: false
+    name: illusive-get-forensics-analyzers
+    outputs:
+    - contextPath: Illusive.Event.incidentId
+      description: The corresponding incident ID
+      type: Number
+    - contextPath: Illusive.Event.eventId
+      description: The corresponding event ID
+      type: Number
+    - contextPath: Illusive.Event.ForensicsAnalyzers.analyzerName
+      description: The forensics analyzer name
+      type: String
+    - contextPath: Illusive.Event.ForensicsAnalyzers.analyzerValue
+      description: The forensics analyzer description
+      type: String
+  - arguments:
+    - default: false
+      description: The event ID to retrieve the triggering process information from
+      isArray: false
+      name: event_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve the triggering process information from Illusive's forensics
+    execution: false
+    name: illusive-get-forensics-triggering-process-info
+    outputs:
+    - contextPath: Illusive.Event.incidentId
+      description: The corresponding incident ID
+      type: Number
+    - contextPath: Illusive.Event.eventId
+      description: The corresponding event ID
+      type: Number
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.commandLine
+      description: The triggering process command line
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.connectionsNum
+      description: The triggering process active connections
+      type: Number
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.md5
+      description: The triggering process md5
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.sha256
+      description: The triggering process sha256
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.name
+      description: The triggering process name
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.parent
+      description: The parent process of the triggering process
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.path
+      description: The triggering process path
+      type: String
+    - contextPath: Illusive.Event.ForensicsTriggeringProcess.startTime
+      description: The triggering process start time
+      type: Date
+  - arguments:
+    - default: false
+      description: The event ID to retrieve the forensics artifacts from
+      isArray: false
+      name: event_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: DESKTOP_SCREENSHOTS
+      description: 'The type of forensics artifact to retrieve '
```

Comment:
If `DESKTOP_SCREENSHOTS` is the only option available and it is also the default value, why do we need this argument?